speech/google/cloud/speech/_gax.py

```diff
@@ -104,9 +104,9 @@ def async_recognize(self, sample, language_code=None,
         audio = RecognitionAudio(content=sample.content,
                                  uri=sample.source_uri)
         api = self._gapic_api
-        response = api.async_recognize(config=config, audio=audio)
+        operation_future = api.async_recognize(config=config, audio=audio)
 
-        return Operation.from_pb(response, self)
+        return Operation.from_pb(operation_future._operation, self)
```

Comment:
`_OperationFuture` has a `last_operation_data` method. I think that's what you're looking for.